cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -29,6 +29,10 @@ spec:
   # Important: if you reinstall the cluster, make sure you delete this directory from each host or else the mons will fail to start on the new cluster.
   # In Minikube, the '/data' directory is configured to persist across reboots. Use "/data/rook" in Minikube environment.
   dataDirHostPath: /var/lib/rook
+  # Wether or not upgrade should continue even if a check fails
```

Comment:
typo: Whether

Reply:
Do we document somewhere what these checks include? Maybe you could add a sentence or two about what those checks are and why it puts you at risk to skip them.

Reply:
Doc added.